tracer/build/_build/Build.GitHub.cs

```diff
@@ -74,6 +74,40 @@ partial class Build
             Console.WriteLine($"PR assigned");
         });
 
+    Target ClosePreviousMilestone => _ => _
```

Comment:
nit: can probably rename this to `CloseMilestone`, this target doesn't have any sense of "previous"

Reply:
done in 670ff1e7ec78fd6da6c207a9d6798fa99b4f630e